tests/openvino/utils_tests.py

```diff
@@ -109,6 +110,7 @@
     "latent-consistency": "echarlaix/tiny-random-latent-consistency",
     "sew": "hf-internal-testing/tiny-random-SEWModel",
     "sew_d": "asapp/sew-d-tiny-100k-ft-ls100h",
+    "snowflake": "katuni4ka/tiny-random-snowflake",
```

Comment:
I think this could be clearer
```suggestion
    "arctic": "katuni4ka/tiny-random-snowflake",
```

Reply:
applied